src/main/java/kdt/prgrms/kazedon/everevent/domain/user/User.java

```diff
@@ -92,4 +93,22 @@ private void checkEmail(String email) {
       throw new InvalidUserArgumentException(ErrorMessage.INVALID_EMAIL_FORMAT, email);
     }
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    User user = (User) o;
+    return Objects.equals(id, user.id);
```

Comment:
이렇게 적절히 줄넘김하는 것은 어떠신가용?? 
```suggestion
    if (this == o) {
      return true;
    }
    
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    
    User user = (User) o;
    return Objects.equals(id, user.id);
```

Reply:
여기두 컨벤션을 적용하는게 맞네용 ㅎㅎ